File: Week 6/pset6/mario/less/mario.py
from cs50 import get_int


def main():
    # Prompt user for height
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break
    # Call the function that prints the piramid
    hashes(height)
    
    
def hashes(height):
    i = 1
    # Add on line
    while i <= height:
        # At every end of the line it will return to the status quo and reassign the variables' values
        # The number of spaces should be the oposit of hashes
        s = height - i
        j = 0
        while s > 0:
            print(" ", end="")
            s -= 1
        while i > j:
            print("#", end="")
            j += 1
        print()
        i += 1
        
        
# Since I'm using custom functions, it's necessary to call main only at the end
main()

File: Week 1/pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    // Prompt user. Hint: do while LOOPS and get_int
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Jump to the other line
    for (int i = 1; i <= n; i++)
    {
        // Add the Spaces
        for (int s = n - 1; i <= s; s--)
        {
            printf(" ");
        }
        // Add the Hashes
        for (int j = 0; i > j; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}